### DISCUSSION TITLE
Working with Windows

### DISCUSSION THREAD

**Discussion by dataDevilABB** [kraiskil/onnx2c#49] [General]
Hey, this maybe a lame question but I have been trying to install this on a windows machine but I am facing many error and I am resolving it one by one but want to ask you whether this will work in a windows machine? Thanks in advance. @kraiskil

**Comment by kraiskil** [comment]
There should not be any fundamental problems building for Windows. That said, it is utterly untested. 
Closest thing to building onnx2c on Windows was by @hakam00ra in issue #33, but seems using a linux VM was the solution there...

None of the code should be Linux specific, that I recall. Including the 3rd party dependencies - everything should be standard C++.  It's the CMakeLists.txt files that are written for GCC (or Clang - there has been a few fixes for building on MacOS along the road). And because of this, the build seems to fail on the VisualStudio compiler.

Maybe it would be easier with mingw or clang on Windows? 

Patches welcome :)
